src/cards/ColorPicker.svelte

```diff
@@ -33,10 +36,16 @@
   cursor: pointer;
   transition: transform 50ms ease-in-out;
 }
-.option:hover {
+
+.option:hover:not(.disabled) {
   transform: scale(1.1);
 }
 
+.disabled {
+  cursor: auto;
+  filter: grayscale(50%);
```

Comment:
this is an accesibilty concern, please make sure you are inclusive to all of us in this community